analysis/_site.yml

```diff
@@ -1,25 +1,28 @@
 name: fucci-seq
-output_dir: "../docs"
-include: [".nojekyll"]
-exclude: ["include"]
+output_dir: ../docs
+include: .nojekyll
+exclude: include
 navbar:
   title: fucci-seq
   left:
-    - text: "Home"
-      href: index.html
-    - text: "About"
-      href: about.html
-    - text: "License"
-      href: license.html
+  - text: Home
+    href: index.html
+  - text: About
+    href: about.html
+  - text: License
+    href: license.html
   right:
-    - icon: fa-github
-      href: https://github.com/jdblischak/workflowr
+  - icon: fa-github
+    href: https://github.com/jdblischak/workflowr
 output:
   html_document:
-    toc: true
-    toc_float: true
+    toc: yes
+    toc_float: yes
     theme: cosmo
     highlight: textmate
     include:
       before_body: include/header.html
       after_body: include/footer.html
+    lib_dir: site_libs
+    self_contained: no
```

Comment:
Why did you add these manually? These should be automatically added by `rmarkdown::render_site()`.

Reply:
I also didn't add these manually... I didn't make changes to`_site.yml`.